bot/exts/fun/off_topic_names.py

```diff
@@ -117,25 +152,133 @@ async def delete_command(self, ctx: Context, *, name: OffTopicName) -> None:
         log.info(f"{ctx.author} deleted the off-topic channel name '{name}'")
         await ctx.send(f":ok_hand: Removed `{name}` from the names list.")
 
-    @otname_group.command(name='list', aliases=('l',))
+    @otname_group.command(name='activate', aliases=('whitelist',))
+    @has_any_role(*MODERATION_ROLES)
+    async def activate_ot_name(self, ctx: Context, name: OffTopicName) -> None:
+        """Whitelist off topic name."""
+        await self.toggle_ot_name_activity(ctx, name, True)
+
+    @otname_group.command(name='deactivate', aliases=('blacklist',))
+    @has_any_role(*MODERATION_ROLES)
+    async def de_activate_ot_name(self, ctx: Context, name: OffTopicName) -> None:
+        """Deactivate/blacklist off topic name."""
+        await self.toggle_ot_name_activity(ctx, name, False)
+
+    @otname_group.command(name='reroll')
+    @has_any_role(*MODERATION_ROLES)
+    async def re_roll_command(self, ctx: Context, ot_channel_index: Optional[int] = None) -> None:
+        """
+        Re-rolls off topic name for an off-topic channel and blacklists the name.
+
+        ot_channel_index: [0, 1, 2, ...]
+        """
+        if ot_channel_index:
+            try:
+                channel = self.bot.get_channel(CHANNELS[ot_channel_index])
+            except IndexError:
+                await ctx.send(f":x: Off-topic channel not found with index {ot_channel_index}.")
```

Comment:
Just a small re-ordering of words:
```suggestion
                await ctx.send(f":x: No off-topic channel found with index {ot_channel_index}.")
```